packages/dynamic-links/ios/RNFBDynamicLinks/RNFBDynamicLinksAppDelegateInterceptor.m

```diff
@@ -99,7 +105,10 @@ - (BOOL)application:(UIApplication *)application
                          @"minimumAppVersion" : dynamicLink.minimumAppVersion == nil
                              ? [NSNull null]
                              : dynamicLink.minimumAppVersion,
-                       }];
+                         @"utmParameters": dynamicLink.utmParametersDictionary == nil
+                             ? @{}
+                             : dynamicLink.utmParametersDictionary,
+                         }];
```

Comment:
To my close examination this looks like a very minor formatting error vs the original post-auto-format
this should fix it
```suggestion
                       }];
```